dev-docs/modules/userid-submodules/lockraim.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: userid
+title: lockr AIM
+description: lockr Alternative Identity Manager sub-module
+useridmodule: lockrAIMIdSystem
+---
+
+Alternative Identity Manager (AIM) is a unified container for identity and data management.
+With AIM’s self-service platform, publishers seamlessly integrate and activate alternative IDs like LiveRamp’s Authenticated Traffic Solution (ATS), Unified ID 2.0 (UID2), ID5 and more. The burden of due diligence and maintenance, coupled with the benefits of server-side calls result in the adoption of multiple alternative IDs, clean rooms like InfoSum and CDPs like Blueconic based on their specific needs.
```

Comment:
Sorry to say, but this description doesn't help me understand exactly what lockr AIM does. Specifically, the last sentence isn't making sense to me: "The burden [...] results in the adoption [...] based on needs." How does AIM solve this problem? Maybe there's a link to a webpage describing it more thoroughly?